### DISCUSSION TITLE
Implement "shared directory" using vite as host application

### DISCUSSION THREAD

**Discussion by ghost** [module-federation/module-federation-examples#4312] [Q&A]
@ScriptedAlchemy , how to use vite as a host application to achieve "shared directory"?
https://github.com/module-federation/module-federation-examples/tree/master/shared-directory

**Comment by ScriptedAlchemy** [comment]
Open issue on vite plugin repo. 
I only maintain the rspack and webpack plugins and maintain the runtime. 
Vite plugin implements the runtime is own, but I personally have not looked at or used vite.

**Comment by ghost** [comment]
@ScriptedAlchemy 

![image](https://github.com/user-attachments/assets/2974ff10-493f-43ae-a786-50cec82d8c42)

Hello, I want to ask if rspack handles shared in remotes during build time, import file from "shared/dir1/file1.js",
Because when I used vite as host, it worked in the development environment, but it didn't work after the build.

**Reply by ScriptedAlchemy** [reply]
everything always works in the rspack versions
